aboutlibraries/src/main/java/com/mikepenz/aboutlibraries/ui/LibsSupportFragment.kt

```diff
@@ -28,4 +30,6 @@ open class LibsSupportFragment : Fragment() {
         libsFragmentCompat.onDestroyView()
         super.onDestroyView()
     }
+
+    override fun getFilter(): Filter = libsFragmentCompat.filter
```

Comment:
This also allows clients that don't use `LibsActivity` to hook up their own search UI